packages/ava/src/advisor/advise-pipeline/advicesForChart.ts

```diff
@@ -4,69 +4,90 @@ import { cloneDeep } from '../utils';
 import { dataToAdvices } from './data-to-advices';
 
 import type { CkbTypes } from '../../ckb';
+import type { Data } from '../../common/types';
 import type { AdviseResult, ChartAdviseParams } from '../types';
 import type { BasicDataPropertyForAdvice, RuleModule } from '../ruler';
 
+export function getDataProps(
```

Comment:
```suggestion
export function filterDataProps(
```

it's confusing that a function named *get* DataProps requires *dataProps* as a parameter.

maybe change the name to `filterDataProps` or `pickDataProps` or sth similar

Reply:
and plz add js-doc for this new func

Reply:
done